app/bundles/PageBundle/Tests/Controller/PageControllerTest.php

```diff
@@ -22,6 +23,75 @@ class PageControllerTest extends MauticMysqlTestCase
      */
     private $id;
 
+    public static function getColumnTests(
```

Comment:
@escopecz  How to deal with method what I can use also on PageController  and EmailController. See examples above.

Reply:
I don't get the question. Can you elaborate?

Reply:
Test are same in two controllers:

- https://github.com/mautic/mautic/pull/10862/files#diff-df7488686a672241a758e3760f10c4dc28f5aec1eb32cb45dc875db25e6ceaf2R65
- https://github.com/mautic/mautic/pull/10862/files#diff-a648d4bc05cfae526c94c208dba1c4e2f1c08188a770adf1279e5c4a9ccdd6c4R130

Could stay like I did it?

Reply:
I see. It is not ideal. It'd be great if you could move this method to a trait in the CoreBundle and then use this trait in more tests.